components/pack/src/PackFile.cpp

```diff
@@ -282,10 +285,28 @@ void PackFile::ExtractTexturesFromBlock()
 		DdsHeader ddsHeader;
 		ddsStream.read(reinterpret_cast<char*>(&ddsHeader), sizeof(DdsHeader));
 
-		// TODO(bwrsandman) the extra sizeof(uint32_t) is unaccounted for
-		if (header.ddsSize - sizeof(ddsHeader) - sizeof(uint32_t) != ddsHeader.pitchOrLinearSize)
```

Comment:
@bwrsandman 
I'm still reading this example but I don't see any checks like that here: https://docs.microsoft.com/en-us/windows/uwp/gaming/complete-code-for-ddstextureloader
I've had to remove this check as the ddsSize field is larger than the pitchOrLinearSize for DXT5 textures.